src/lustre/lustreExpr.ml

```diff
@@ -233,6 +233,12 @@ let pre_offset = Numeral.(pred cur_offset)
 (* Pretty-printing                                                        *)
 (* ********************************************************************** *)
 
+(** Pretty-print a bound or fixed annotation *)
+let pp_print_bound_or_fixed pp ppf = function
+  | Bound x -> pp ppf x
+  | Fixed x -> pp ppf x
+  | Unbound x -> pp ppf x
```

Comment:
It seems that `pp` is always `LustreExpr.pp_print_expr`. Use that function directly.